src/grid-list/grid-list.jsx

```diff
@@ -84,7 +84,7 @@ const GridList = React.createClass({
       const childRows = currentChild.props.rows || 1;
       const itemStyle = this.mergeStyles(styles.item, {
         width: (100 / cols * childCols) + '%',
-        height: cellHeight * childRows + padding,
+        height: parseInt(cellHeight) * childRows + padding,
```

Comment:
Sorry it's my fault (I read `PropTypes.stringOrNumber` instead of `PropTypes.number`). In this case we need to change the default value of `cellHeight` from `cellHeight: '180px'` to `cellHeight: 180` to avoid the `NaN` of `cellHeight * childRows + padding`